amazon-corretto-crypto-provider.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: amazon-corretto-crypto-provider
+  version: 2.3.1
+  epoch: 0
+  description: |
+    The Amazon Corretto Crypto Provider (ACCP) is a collection of high-performance cryptographic implementations exposed via the standard JCA/JCE interfaces.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - build-base
+      - bash
+      - cmake
+      - clang-16-extras
+      - ca-certificates-bundle
+      - openjdk-11
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/corretto/amazon-corretto-crypto-provider
+      expected-commit: 296f3821ee8436ea8a311709c0a89e198b19466c
+      tag: ${{package.version}}
+      recurse-submodules: true
+
+  - working-directory: /home/build/aws-lc
+    pipeline:
+      - uses: patch
+        with:
+          patches: /home/build/aws-lc.patch
+
+  - runs: |
+      export JAVA_HOME=/usr/lib/jvm/java-11-openjdk
+      export CFLAGS="-Wno-error=ignored-attributes"
+      ./gradlew build
+
+  - runs: |
+      mkdir -p ${{targets.destdir}}/usr/share/java/amazon-corretto-crypto-provider
+      mv build/lib/AmazonCorrettoCryptoProvider.jar ${{targets.destdir}}/usr/share/java/amazon-corretto-crypto-provider/AmazonCorrettoCryptoProvider.jar
+
+update:
+  enabled: true
+  github:
+    identifier: corretto/amazon-corretto-crypto-provider
+    use-tag: true
```

Comment:
I think we should let it use releases instead.

Reply:
Thanks @stormqueen1990 , just changed it